src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -645,7 +646,7 @@ public function create_child_record() {
 	 * @param array $meta Meta to add to the post
 	 */
 	public function maybe_add_meta_via_pre_wp_44_method( $id, $meta ) {
-		if ( -1 !== version_compare( get_bloginfo( 'version' ), '4.4' ) ) {
+		if ( - 1 !== version_compare( get_bloginfo( 'version' ), '4.4' ) ) {
```

Comment:
A space between `-` and a number only makes sense when you are writing an equation (e.g. `5 - 1`). When writing a single negative number, it should be written `-1`. To reinforce my statement, simply run `number_format( - 1 );` and you'll see that the result is a proper `-1` with no space.

```suggestion
		if ( -1 !== version_compare( get_bloginfo( 'version' ), '4.4' ) ) {
```